scripts/src/report/report.py

```diff
@@ -41,6 +42,10 @@ def render(query, bold=False):
     rendered_html = template.render(**context, **static_files)
 
     # TODO: If BOLD, replace 'identity' with 'similarity'
+    if bold:
+        rendered_html = re.sub(r"\bidentity\b", "Similarity",
+                               rendered_html,
+                               flags=re.IGNORECASE)
```

Comment:
Probably better to do two of these and preserve case e.g. `identity -> similarity`